src/db/contractTransactionReceipts/createContractTransactionReceipts.ts

```diff
@@ -1,36 +1,19 @@
+import { ContractTransactionReceipts, Prisma } from "@prisma/client";
 import { PrismaTransaction } from "../../schema/prisma";
 import { getPrismaWithPostgresTx } from "../client";
 
-interface ContractTransactionReceiptEntry {
-  chainId: number;
-  blockNumber: number;
-  contractId: string;
-  contractAddress: string;
-  transactionHash: string;
-  blockHash: string;
-  timestamp: Date;
-  to: string;
-  from: string;
-  transactionIndex: number;
-  value: string;
-  data: string;
-  gasUsed: string;
-  effectiveGasPrice: string;
-  status: number;
-}
-
 export interface BulkInsertContractLogsParams {
   pgtx?: PrismaTransaction;
-  txReceipts: ContractTransactionReceiptEntry[];
+  receipts: Prisma.ContractTransactionReceiptsCreateInput[];
 }
 
 export const bulkInsertContractTransactionReceipts = async ({
   pgtx,
-  txReceipts,
-}: BulkInsertContractLogsParams) => {
+  receipts,
+}: BulkInsertContractLogsParams): Promise<ContractTransactionReceipts[]> => {
   const prisma = getPrismaWithPostgresTx(pgtx);
-  return await prisma.contractTransactionReceipts.createMany({
-    data: txReceipts,
+  return await prisma.contractTransactionReceipts.createManyAndReturn({
```

Comment:
`createManyAndReturn` bulk inserts N rows and returns the resulting rows. This is very useful in order to rely on using the Prisma type.